addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.js

```diff
@@ -12,6 +12,7 @@ patch(SlideUploadCategory.prototype, {
         super.setup();
         this.state.choices.certifications = [];
         this.state.choices.certificationId = "";
+        this.state.choices.isError = false;
```

Comment:
Technically this has nothing to do with "choices".

Can we simply do "this.state.showCertificationRequiredError" ?